packages/orbit-components/src/Itinerary/Itinerary.stories.tsx

```diff
@@ -822,9 +803,6 @@ export const MultipleBanners = () => {
 };
 
 export const RTL = () => {
```

Comment:
Some elements of this story look off.

This branch:
![image](https://github.com/kiwicom/orbit/assets/8863238/c384f229-8afc-4722-b7ec-208ce97764ff)

Current master:
![image](https://github.com/kiwicom/orbit/assets/8863238/080076f9-3cad-4dd0-a102-4c7916485b79)

Reply:
Thank, well spotted. In the past, we had an automation in Styled Components that would convert left to right in RTL. They should now be replaced by start and end, but in some of them I failed to do so. Fixed.